eq-author/src/App/page/Design/MovePageModal/index.test.js

```diff
@@ -89,3 +99,72 @@ describe("MovePageModal: questionnaire not loaded", () => {
     expect(wrapper.isEmptyRender()).toBe(true);
   });
 });
+describe("MovePageModal: buildPageList", () => {
+  let output, folders;
+
+  beforeEach(() => {
+    folders = [
+      {
+        id: "folder-1",
+        enabled: true,
+        position: 0,
+        __typename: "Folder",
+        pages: [
+          {
+            id: "folder-1-page-1",
+            position: 0,
+          },
+          {
+            id: "folder-1-page-2",
+            position: 1,
+          },
+        ],
+      },
+      {
+        id: "folder-2",
+        enabled: false,
+        position: 1,
+        pages: [
+          {
+            id: "folder-2-page-1",
+            position: 2,
+          },
+        ],
+      },
+    ];
+    output = buildPageList(folders);
+  });
+
+  it("should include enabled folders in output", () => {
+    expect(
+      output.filter(({ __typename }) => __typename === "Folder")[0]
+    ).toMatchObject({
+      __typename: "Folder",
+      parentEnabled: false,
+      enabled: true,
+    });
+  });
+
+  it("should not included disabled folders in output", () => {
+    expect(
+      output.filter(({ enabled, type }) => !enabled && type === "Folder")
+    ).toHaveLength(0);
+  });
+
+  it("should include parent details on enabled folders", () => {
+    expect(output.filter(({ id }) => id.startsWith("folder-1-page"))).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          parentEnabled: true,
+          parentId: output[0].id,
+          position: expect.any(Number),
+        }),
+      ])
+    );
+  });
```

Comment:
```suggestion
  it("should include parent details on enabled folders", () => {
  
    const enabledFolders = output.filter(({ id }) => id.startsWith("folder-1-page"));
    expect(enabledFolders).toEqual(
      expect.arrayContaining([
        expect.objectContaining({
          parentEnabled: true,
          parentId: output[0].id,
          position: expect.any(Number),
        }),
      ])
    );
  });
```

The assertion of this test is slightly longer than the last, so you could toss the enabled folders into a variable to make it a little nicer on the eye.